frontend/src/components/common/TooltipWrapper.tsx

```diff
@@ -30,16 +30,19 @@ interface ITooltipWrapperProps extends Partial<OverlayTriggerProps> {
 const TooltipWrapper: React.FunctionComponent<ITooltipWrapperProps> = props => {
   return (
     <>
-      {props.toolTip?.length ? (
```

Comment:
This conditional render was causing input fields to lose focus when they shifted from an in error to not in error state.